packages/nx-plugin/src/generators/component/generator.ts

```diff
@@ -15,14 +10,20 @@ export default async function (tree: Tree, options: ComponentGeneratorSchema) {
   const { root: projectRoot } = project;
   const paths = getPackagePaths(workspaceRoot, projectRoot);
 
-  generateFiles(tree, path.join(__dirname, 'files'), projectRoot, {...options, npmScope});
+  generateFiles(tree, path.join(__dirname, 'files'), projectRoot, {
+    ...options,
+    npmScope,
+  });
 
   let main = tree.read(paths.main)?.toString();
   if (!main) {
     throw new Error('No index.ts main entrypoint');
   }
 
-  main = `export * from './components/${componentName}';` + '\n' + main;
+  main =
```

Comment:
we could also run `eslint --fix` which would produce the same output, which for more complicated use-case is the way to go. for this use case this is good enough and won't slow down generator execution by spawning another process